app/src/main/res/values/styles_v2.xml

```diff
@@ -713,4 +713,17 @@
     <item name="android:paddingStart">@dimen/spacing_16</item>
     <item name="android:paddingTop">@dimen/spacing_16</item>
   </style>
+
+  <style name="Clinic.V2.TextInputLayout.FilledBox" parent="Widget.MaterialComponents.TextInputLayout.FilledBox">
+    <item name="hintAnimationEnabled">false</item>
```

Comment:
On the emulator, the animation was a little bit choppy. Since most of our users might have a low to mid-end devices it's better to not have it maybe. WDYT?